app/models/miq_worker.rb

```diff
@@ -219,6 +219,10 @@ def worker_settings(options = {})
     self.class.fetch_worker_settings_from_server(miq_server, options)
   end
 
+  def heartbeat_file
+    ENV["WORKER_HEARTBEAT_FILE"] || Rails.root.join("tmp", "#{guid}.hb")
```

Comment:
No issue other just having it clean up after itself. Every time a worker is started it will get a new guid and a new file so they could accumulate fast.

I think you got pinged because you previously added comments.